app/src/main/java/com/example/peterchu/watplanner/scheduler/ScheduleUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.example.peterchu.watplanner.scheduler;
+
+import android.graphics.Color;
+
+import com.alamkanak.weekview.WeekViewEvent;
+import com.example.peterchu.watplanner.Models.Schedule.CourseComponent;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class ScheduleUtils {
+
+    private ScheduleUtils() {
+        // Util class
```

Comment:
actually you don't need this do you? This will be the same as default constructor (I think)